### PR TITLE
WPF-61114 Prepared UG content for SmithChart

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
Hi @vigneshrameshsync ,
                   I have prepared UG content for SmiithChart.

### CODE REVIEW

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

**Comment by TamilarasanGunasekaran** [comment]
rebuild

File: uwp/Smith-Chart/Series.md [hunk 1]
```diff
@@ -144,3 +144,31 @@ chart.Series.Add(series2);
 {% endtabs %}
 
 ![SfSmithChart Series Visibility](Series_images/Series_img3.png)
+
+## Customizing Series Data
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply uwp/Smith-Chart/Series.md]: I have made changes as you mentioned.

**Review by vigneshrameshsync — APPROVED** [review]
The content is fine.

File: uwp/Smith-Chart/Series.md [hunk 2]
```diff
@@ -144,3 +144,29 @@ chart.Series.Add(series2);
 {% endtabs %}
 
 ![SfSmithChart Series Visibility](Series_images/Series_img3.png)
+
+## Data plotting customization
+
+By default, the data points are plotted by sorting them based on the resistance values and it can be plotted based on their index value as it is by setting the [`ArrangeByIndex`]() property as true.
```
Comment (Chrispineagunja) [inline uwp/Smith-Chart/Series.md:150]: By default, the data points are plotted by sorting them based on the resistance values and they can be plotted based on their index values as it is by setting the [`ArrangeByIndex`]() property as true.

**Comment by TamilarasanGunasekaran** [comment]
rebuild

File: uwp/Smith-Chart/Series.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply uwp/Smith-Chart/Series.md]: I have updated.

**Comment by TamilarasanGunasekaran** [comment]
rebuild

**Pull request merged**